cg/cli/deliver/base.py

```diff
@@ -133,6 +133,54 @@ def deliver_ticket(
     )
 
 
+@deliver.command(
+    name="sample",
+    help="Deliver fastq or bam files for a sample to the customer inbox on the HPC "
+    "and start an Rsync job to clinical-delivery. "
+    "NOTE: the dry-run flag will copy files to the customer inbox on Hasta, "
+    "but will not perform the Rsync job.",
+)
+@click.pass_obj
+@click.option(
+    "-c",
+    "--case-id",
+    required=True,
+    help="The case the sample is on.",
+)
+@click.option("-s", "--sample-id", required=True, help="Deliver the files for a specific sample.")
+@click.option(
+    "-d",
+    "--delivery-type",
+    required=True,
+    help="The delivery type to use.",
+    type=click.Choice(choices=["fastq", "bam"]),
+)
+@DRY_RUN
+def deliver_sample_raw_data(
+    context: CGConfig,
+    case_id: str,
+    sample_id: str,
+    delivery_type: FileDeliveryOption,
+    dry_run: bool,
+):
+    """
+    Deliver fastq or bam files for a single sample to the customer inbox on the HPC
+    """
+    inbox: str = context.delivery_path
+    service_builder: DeliveryServiceFactory = context.delivery_service_factory
+    case: Case = context.status_db.get_case_by_internal_id(internal_id=case_id)
+    if not case:
+        LOG.error(f"Could not find case with id {case_id}")
+        return
```

Comment:
```suggestion
        click.Abort
```

Reply:
it is a manual command here an error will be logged not sure if click abort adds any value